app/controllers/application_controller/ci_processing.rb

```diff
@@ -853,7 +879,7 @@ def set_discover_title(type, controller)
     else
       controller_table = ui_lookup(:tables => controller)
       if controller == "ems_cloud"
-        return "Amazon #{controller_table}"
+        return "#{controller_table}"
```

Comment:
@lgalis both if/else branch is returning same thing so we should just do ``return  ui_lookup(:tables => controller)``